prog/dftb+/prg_dftb/mainio.F90

```diff
@@ -2373,8 +2374,7 @@ subroutine writeDetailedOut1(fd, fileName, tAppendDetailedOut, iDistribFn, nGeoS
     integer :: iAt, iSpin, iEgy, iK, iSp, iSh, iOrb, kk
     logical :: tSpin
 
-    character(*), parameter :: formatEigen = "(F14.8)"
-    character(*), parameter :: formatFilling = "(F12.5)"
+    character(*), parameter :: formatEigen = "(F14.8, F9.5)"
```

Comment:
What about constraining the number of digits to yield meV precision only? I find too many digits (beyond DFTB precision) disturbing when analysing results.

Reply:
Will do.
But on the other hand, for finite difference derivatives, 3 decimal places is not enough. So in the longer term, a user flag to set the precision level might be useful.